integrations/access/opsgenie/client.go

```diff
@@ -251,6 +257,23 @@ func (og Client) CheckHealth(ctx context.Context) error {
 		return trace.Wrap(err)
 	}
 	defer resp.RawResponse.Body.Close()
+
+	if og.StatusSink != nil {
+		var code types.PluginStatusCode
+		switch {
+		case resp.StatusCode() == http.StatusUnauthorized:
+			code = types.PluginStatusCode_UNAUTHORIZED
+		case resp.StatusCode() >= 200 && resp.StatusCode() < 400:
+			code = types.PluginStatusCode_RUNNING
+		default:
+			code = types.PluginStatusCode_OTHER_ERROR
+		}
+		if err := og.StatusSink.Emit(ctx, &types.PluginStatusV1{Code: code}); err != nil {
+			log := logger.Get(resp.Request.Context())
```

Comment:
Where does this logger come from? Is this the default one? If so, it might be clearer to get the default logger directly.